services/portfolios_ca/src/services/targets.ts

```diff
@@ -0,0 +1,97 @@
+export default () => {
+  const Targets = {
+    /**
+     * Get values of all portfolios for a specific target within a given area
```

Comment:
Update this description and the return value, now this function includes target basic information

Reply:
Done